apps/web/README.MD

```
@@ -21,6 +21,7 @@ This is the **recommended** deployment option for **Gemini Proxy**. It's a full-
 
 - [Environment Variables](#️-environment-variables)
 - [Vercel Deployment](#-vercel-deployment-recommended)
+- [Cloudflare Deployment](#-cloudflare-deployment-opennext)
```

markdownlint-cli2 (0.23.2)

[warning] Link fragments should be valid

(MD051, link-fragments)